client/lib/query-manager/theme/key.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import omitBy from 'lodash/omitBy';
+
+/**
+ * Internal dependencies
+ */
+import PaginatedQueryKey from '../paginated/key';
+import { DEFAULT_THEME_QUERY } from './constants';
+
+/**
+ * Returns true if the specified key value query pair is identical to that of
+ * the default theme query, is null, or is undefined.
+ *
+ * @param  {*}       value Value to check
+ * @param  {String}  key   Key to check
+ * @return {Boolean}       Whether key value matches default query or is null
+ */
+function isDefaultOrNullQueryValue( value, key ) {
+	return (
+		null == value || // Double-equals null checks undefined, null
```

Comment:
Even though I started with it, the double-equals feels very non-obvious (to the point of requiring a comment explaining its behavior). Also, would a general truthiness check here suffice? I think with the other implementations I had worried that empty string or `0` could have an impact on query results, but that's not necessarily always the case.